Update Holacracy-Constitution.md

osmolkov opened the pull request:
I Propose to Update Russuan translation due to changes in the original (Commit: fefca784f8f1d31401bfb8a594fcb99f9d22e90a)

osmolkov committed:
Update Holacracy-Constitution.md

Update translation due to changes in the original
Commit: fefca784f8f1d31401bfb8a594fcb99f9d22e90a

osmolkov committed:
Rename Holacracy-Constitution.md to Конституция-Холакратии.md

osmolkov commented:
@Jen-Sposit, please merge my pull request.

Jen-Sposit commented:
@osmolkov, it looks like I'm unable to accept the pull request due to a conflict highlighted on lines 85-89. I'm unsure how to fix it because I don't know what may mess up the translation. Can you take a look? I can also add you as an editor if you like, so you can make direct changes to the text.

osmolkov committed:
Update Конституция-Холакратии.md

osmolkov committed:
Update Конституция-Холакратии.md пункт 1.2.5

alextsyglin committed:
1.1 убрал слова "операционной деятельности" т.к. этого нет в первоисточнике

1.1 
Роль также может содержать "Политики" — то есть ограничения или расширение полномочий, или специальные правила, применяемые к операционной деятельности в рамках Роли.
A Role may also hold “Policies”, which are grants or constraints of authority, or special rules that apply within that Role.

Убрал слова Операционной дятельности

alextsyglin committed:
1.3.3 Убрал в Якорном круге "и нет Обязательств" - этого нет в первоисточнике

alextsyglin committed:
1.3.5 Фасилитатор и секретарь - перефразировал

Было:
Выбранный Фасилитатор выполняет "Роль Фасилитатора" в Круге с миссией "Законодательство Круга и операционные практики соответствуют Конституции".

Стало:
"Законодательные и операционные практики соответствуют конституции"

alextsyglin committed:
1.2.3 Исправил опечатку  в слове "Следующих Действий"

alextsyglin committed:
1.3.5 Добавил "возложенных на них этой Конституцией" как в первоисточнике

Было:
Никакой Круг не может изменить или убрать Миссию, Домены или Обязательства этих Ролей, или удалить сами Роли.

Стало:
Никакой Круг не может изменить или убрать Миссию, Домены или Обязательства этих Ролей возложенных на них этой Конституцией, или удалить сами Роли.

alextsyglin committed:
5.1.1 Исправил опечатку букву "а"

Было: 
Выбранный Представитель Круга становится Членом любого Круга, содержащего этот Круга

Стало:
Выбранный Представитель Круга становится Членом любого Круга, содержащего этот Круг
